README.md

```diff
@@ -1,120 +1,31 @@
-# Map Colonies typescript service template
+# Job Manager Server
 
-----------------------------------
+This server provides a management interface for job trees and includes a RESTful API for interacting with a PostgreSQL database.
 
-This is a basic repo template for building new MapColonies web services in Typescript.
+**Key Features:**
 
-> [!IMPORTANT]
-> To regenerate the types on openapi change run the command `npm run generate:openapi-types`.
+* Manages job trees.
+* Provides a RESTful API for database management.
 
-## Development
-When in development you should use the command `npm run start:dev`. The main benefits are that it enables offline mode for the config package, and source map support for NodeJS errors.
+**Requirements:**
 
-### Template Features:
+* **PostgreSQL:** Version 13 or higher is required.
 
-- eslint configuration by [@map-colonies/eslint-config](https://github.com/MapColonies/eslint-config)
+------------------------
 
-- prettier configuration by [@map-colonies/prettier-config](https://github.com/MapColonies/prettier-config)
+## Migration
+**Development Workflow:**
```

Comment:
1. title for this is "how to create a new migration".
2. its missing some starting sentence for context.
3. also missing how to just run the exiting migrations for starting development: https://www.prisma.io/docs/orm/prisma-migrate/workflows/development-and-production#development-environments

Reply:
done